VideoWeb/VideoWeb.Common/Configuration/AzureAdConfiguration.cs

```diff
@@ -1,9 +1,13 @@
+using System.Configuration;
+
 namespace VideoWeb.Common.Configuration
 {
     public class AzureAdConfiguration
     {
-        public string ClientId { get; set; }
-        public string ClientSecret { get; set; }
+        public string ClientIdV2 { get; set; }
```

Comment:
Why do we have a ClientIdV2 property now?

Reply:
we need the new app registrations to make this work but if we change over the existing it will break.. actually now thinking about it more we could change the pipeline maybe to point KV v2 value at clientid and that would work for rolling back and not breaking future ones

Reply:
changed :)